addons/pinniped/config-controller/manifests/role.yaml

```diff
@@ -18,6 +18,7 @@ rules:
   resources:
   - secrets
```

Comment:
any chance we need `create` as well on secrets for the v1alpha1 case?

Reply:
yup, added!